the_mines/download/get_html.py

```diff
@@ -5,6 +5,35 @@
 logger = logging.getLogger("app")
 
 
+def download_raw_html(url):
+    """Download html page data
+    Args:
+        url (str): target url to download
+    Returns:
+        raw html string output
+    """
+    try:
+        logger.debug("Downloading data file")
+        data = str(requests.get(url).content)
+
+    except Exception:
+        logger.debug(f"Unable to download file using params: {matchday}, {season}")
+
+    return data
+
+
+def download_default_html():
+    try:
+        logger.debug("Downloading data file")
+        url = "https://www.fussballdaten.de/bundesliga/"
+        data = str(requests.get(url).content)
+
+    except Exception:
+        logger.debug(f"Unable to download file")
+
+    return data
```

Comment:
This code is basically a duplicate of what is done above without a param option.  Just pass `"https://www.fussballdaten.de/bundesliga/"` to the method above and you'll get the same result.